src/utils/checkTypes.js

```diff
@@ -1,10 +1,12 @@
 import invariant from 'invariant'
 import isPlainObject from './isPlainObject'
 
-function typecheck(type, name, obj) {
+function typecheck(types, name, obj) {
   invariant(
-    typeof obj === type,
-    `${name} must be a ${type}. Instead received a %s.`,
+    Array.isArray(types)
+      ? types.some(t => typeof obj === t)
+      : typeof obj === types,
+    `${name} must be a ${types}. Instead received a %s.`,
```

Comment:
yeah, it would probably read better that way. (Of course this really isn't super important since it is only something shown in the dev console...)